scripts/windows/production_test.ps1

```diff
@@ -0,0 +1,8 @@
+$ErrorActionPreference = "Stop"
+Remove-Item kjudge.db*
+
+& "scripts\windows\production_build.ps1"
+
+Invoke-Expression ".\kjudge $args"
+
+# pwsh -c scripts/windows/production_test.ps1 --sandbox=raw to run this test script
```

Comment:
Interesting. I wonder if we can run some CI on this? 
Running with `$args = --help` (which spawns the help message and quits immediately) would do.

Reply:
Since this fails if we run the executable without a database file already created, I think we can have something like --dry-run, which tries to create a temporary database file, and run other tests too.

Reply:
> Since this fails if we run the executable without a database file already created

This sounds wrong. Can you post the error? Could be a bug.

Reply:
I don't think it's something nasty. It's just that when the program creates a new database file, the SQL assets are accessed.

Reply:
That should be ok, no?

Reply:
If SQL assets are accessed, then the embed file system is used and that triggers the error.

Reply:
That's what we were fixing with this PR right?

Reply:
Yeah that's right